test-result-summary-client/src/Build/Output/Output.jsx

```diff
@@ -28,6 +29,10 @@ export default class Output extends Component {
         let data = {};
         const { id } = getParams( this.props.location.search );
         if ( outputType === "test" ) {
+<<<<<<< HEAD
+            const info = await fetchData(`/api/getTestById?id=${id} `);
+            const result = await fetchData(`/api/getOutputById?id=${info.testOutputId}`);
+=======

```

Comment:
Please fix this.

Reply:
Fixed. Don't know how I missed that :sweat_smile: